apps/web-client/src/lib/db/cr-sqlite/__tests__/reconciliation-order.test.ts

```diff
@@ -1321,3 +1323,204 @@ describe("Misc helpers", () => {
 		expect(result["Supplier 2"]).toHaveLength(1);
 	});
 });
+describe("Reconciliation order deletion", () => {
+	describe("deleteReconciliationOrder", () => {
+		it("should delete a reconciliation order and its associated lines", async () => {
+			const db = await getRandomDb();
+
+			await addBooksToCustomer(db, 1, ["1"]);
+			await createSupplierOrder(db, 1, 1, [{ isbn: "1", quantity: 1, supplier_id: 1 }]);
+
+			const [{ id: supplierOrderId }] = await getPlacedSupplierOrders(db);
+
+			// Create reconciliation order with lines
+			const reconciliationOrderId = 123;
+			await createReconciliationOrder(db, reconciliationOrderId, [supplierOrderId]);
+			await addOrderLinesToReconciliationOrder(db, reconciliationOrderId, [
+				{ isbn: "1", quantity: 2 },
+				{ isbn: "2", quantity: 1 }
+			]);
+
+			// Delete the reconciliation order
+			await deleteReconciliationOrder(db, reconciliationOrderId);
+
+			// Verify order is deleted
+			expect(await getReconciliationOrder(db, reconciliationOrderId)).toBeUndefined();
+			// Verify lines are deleted
+			await expect(getReconciliationOrderLines(db, reconciliationOrderId)).rejects.toThrow(
```

Comment:
This just tells us the `reconciliation_order` has been deleted. We should also check the `reconciliation_order_line` table isn't returning anything